tests/end-to-end/upgrade/chart/upgrade/templates/test/test-upgrade.yaml

```diff
@@ -23,4 +24,22 @@ spec:
       value: "info"
     - name: DOMAIN
       value: "{{ .Values.global.domainName }}"
+  - image: eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20190325-ff66a3a
```

Comment:
Here, this approach is a bit dangerous. Incase the test pod is slow in ContainerCreation process, we will kill istio-proxy. When the test pod runs the tests, everything will fail.
Why don't we apply this approach https://github.com/kyma-project/kyma/pull/3385/files#diff-da0dbb74a6bb4540d5dc5fadfa5913c9R54?